### ISSUE TITLE
[Accessibility] Carousel block - Navigation buttons are not highlighted when using Tab key

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#3252]
**Describe the bug**
- When clicking Tab key and trying to navigation to Carousel blocks, it doesn't focus/highlight
- Other note: Keyboard focus works if using VoiceOver
   

**To Reproduce**
1. Add a new page
2. Add a Carousel block
3. Add a Button block before and after the Carousel block (optional; only to check if keyboard focus is working)
4. Save page and view frontend
5. Press Tab key
6. See that it doesn't focus on the Carousel navigation buttons

**Expected behavior**
- Navigation buttons should be selected when clicking Tab key

**Screenshots**

https://github.com/user-attachments/assets/2b0aca1a-5c34-46e7-9724-0cbef6f7cb67

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Browser Version: Version 126.0.6478.183 (Official Build) (arm64)
